src/library/trackcollection.cpp

```diff
@@ -23,11 +23,6 @@ TrackCollection::TrackCollection(
           m_trackDao(m_cueDao, m_playlistDao,
                      m_analysisDao, m_libraryHashDao, pConfig) {
     // Forward signals from TrackDAO
-    connect(&m_trackDao,
-            &TrackDAO::trackClean,
-            this,
-            &TrackCollection::trackClean,
-            /*signal-to-signal*/ Qt::DirectConnection);
```

Comment:
Was that the root cause of the UI freeze?

Reply:
No, that was just unused code. I comment inline.